ndnfs/driver/driver.go

```diff
@@ -0,0 +1,360 @@
+	    log.Info("Error running mkdir command: ", err, "{", string(out), "}")
+	}
+	log.Debug(DN, "Mounting Volume ", r.Name)
+	args := []string{"-t", "nfs", nfs, mnt}
+	if out, err := exec.Command("mount", args...).CombinedOutput(); err != nil {
+		log.Error("Error running mount command: ", err, "{", string(out), "}")
+		err = errors.New(fmt.Sprintf("%s: %s", err, out))
+		if err != nil {
+			return volume.Response{Err: err.Error()}
+		}
+	}
+	return volume.Response{Mountpoint: mnt}
+}
+
+func (d NdnfsDriver) Path(r volume.Request) volume.Response {
+	log.Info(DN, "Path volume: ", r.Name, " Options: ", r.Options)
+	mnt := fmt.Sprintf("%s%s", d.Config.Mountpoint, r.Name)
+	return volume.Response{Mountpoint: mnt}
+}
+
+func (d NdnfsDriver) Remove(r volume.Request) volume.Response {
+	log.Info(DN, "Remove volume: ", r.Name, " Options: ", r.Options)
+	d.Mutex.Lock()
+	defer d.Mutex.Unlock()
+	nfsList, err := d.GetNfsList()
+	if err != nil {
+		log.Info("Error getting nfs list", err)
+	}
+	var path, service string
+	for i := range(nfsList) {
+		if strings.Contains(nfsList[i], r.Name) {
+			path = nfsList[i]
+		}
+	}
+	if os.Getenv("CCOW_SVCNAME") != "" {
+		service = os.Getenv("CCOW_SVCNAME")
+	} else {
+		service = d.Config.Servicename
+	}
+	data := make(map[string]interface{})
+	data["serve"] = path
+	url := fmt.Sprintf("service/%s/serve", service)
+	_, err = d.Request("DELETE", url, data)
+	if err != nil {
+		log.Info("Error while handling request", err)
+	}
+
+	parts := strings.Split(path, "/")
+	url = fmt.Sprintf("clusters/%s/tenants/%s/buckets/%s", parts[0], parts[1], parts[2])
+	_, err = d.Request("DELETE", url, nil)
+
+	mnt := filepath.Join(d.Config.Mountpoint, r.Name)
+	if out, err := exec.Command("rm", "-rf", mnt).CombinedOutput(); err != nil {
+		log.Info("Error running rm command: ", err, "{", string(out), "}")
+	}
+
+	return volume.Response{}
+}
+
+func (d NdnfsDriver) Unmount(r volume.UnmountRequest) volume.Response {
+	log.Info(DN, "Unmount volume: ", r.Name)
+	d.Mutex.Lock()
+	defer d.Mutex.Unlock()
+	nfs := fmt.Sprintf("%s/%s", d.Config.Mountpoint, r.Name)
+	if out, err := exec.Command("umount", nfs).CombinedOutput(); err != nil {
+		log.Error("Error running umount command: ", err, "{", string(out), "}")
+	}
+	return volume.Response{}
+}
+
+func (d NdnfsDriver) GetNfsList() (nfsList []string, err error) {
+	var body []byte
+	if os.Getenv("CCOW_SVCNAME") != "" {
+		body, err = d.Request(
+			"GET", fmt.Sprintf("service/%s", os.Getenv("CCOW_SVCNAME")), nil)
+	} else {
+		body, err = d.Request(
+			"GET", fmt.Sprintf("service/%s", d.Config.Servicename), nil)
+	}
+			
+	r := make(map[string]map[string]map[string]interface{})
+	jsonerr := json.Unmarshal(body, &r)
+	if (jsonerr != nil) {
+		log.Error(jsonerr)
+	}
+	if r["response"]["data"]["X-Service-Objects"] == nil {
+		return
+	}
+	strList := strings.Trim((r["response"]["data"]["X-Service-Objects"].(string)), "[]")
+	nfsList = strings.Split(strList, ",")
```

Comment:
when was this introduced? In my env, I have the response like so:
"X-Service-Objects":"[\"cl/tn/n3\"]"